db/migrate/20171106143927_downcase_resource_group_ems_ref_for_azure.rb

```diff
@@ -0,0 +1,14 @@
+class DowncaseResourceGroupEmsRefForAzure < ActiveRecord::Migration[5.0]
+  class ResourceGroup < ActiveRecord::Base; end
+
+  def up
+    ResourceGroup.all.each do |resource_group|
+      if resource_group.ems_ref =~ /^\/subscriptions\/.*?\/resourcegroups\/\w+/i
+        resource_group.update_attributes(
+          :ems_ref => resource_group.ems_ref.downcase,
+          :type    => 'ManageIQ::Providers::Azure::ResourceGroup'
```

Comment:
This needs to be set on *ALL* resource groups and the current logic prevents that...I recommend making this a separate data migration that runs prior to this one.

Reply:
I'm afraid that I don't understand. I wasn't totally sure if any other providers were using this. If they are, it would botch those records, wouldn't it?

Reply:
I believe that `resource_groups` was added specifically for Azure.  So, no other provider should be using it.